examples/oft-solana/tasks/solana/index.ts

```diff
@@ -107,3 +135,79 @@ export const getLayerZeroScanLink = (hash: string, isTestnet = false) =>
 
 export const getExplorerTxLink = (hash: string, isTestnet = false) =>
     `https://explorer.solana.com/tx/${hash}?cluster=${isTestnet ? 'devnet' : 'mainnet-beta'}`
+
+export const getAddressLookupTable = async (connection: Connection, umi: Umi, fromEid: EndpointId) => {
+    // Lookup Table Address and Priority Fee Calculation
+    const lookupTableAddress = LOOKUP_TABLE_ADDRESS[fromEid]
+    assert(lookupTableAddress != null, `No lookup table found for ${formatEid(fromEid)}`)
+    const addressLookupTableInput: AddressLookupTableInput = await fetchAddressLookupTable(umi, lookupTableAddress)
+    if (!addressLookupTableInput) {
+        throw new Error(`No address lookup table found for ${lookupTableAddress}`)
+    }
+    const { value: lookupTableAccount } = await connection.getAddressLookupTable(toWeb3JsPublicKey(lookupTableAddress))
+    if (!lookupTableAccount) {
+        throw new Error(`No address lookup table account found for ${lookupTableAddress}`)
+    }
+    return {
+        lookupTableAddress,
+        addressLookupTableInput,
+        lookupTableAccount,
+    }
+}
+
+export const getComputeUnitPriceAndLimit = async (
+    connection: Connection,
+    ixs: Instruction[],
+    wallet: KeypairSigner,
+    lookupTableAccount: AddressLookupTableAccount
+) => {
+    const { averageFeeExcludingZeros } = await getFee(connection)
+    const priorityFee = Math.round(averageFeeExcludingZeros)
+    const computeUnitPrice = BigInt(priorityFee)
+
+    const computeUnits = await getSimulationComputeUnits(
+        connection,
+        ixs.map((ix) => toWeb3JsInstruction(ix)),
+        toWeb3JsPublicKey(wallet.publicKey),
+        [lookupTableAccount]
+    )
+
+    if (!computeUnits) {
+        throw new Error('Unable to compute units')
+    }
+
+    return {
+        computeUnitPrice,
+        computeUnits,
+    }
+}
+
+export const addPerformanceInstructions = async (
```

Comment:
can we call them `addComputeUnitInstructions` instead? It's more descriptive that way.

Reply:
I agree, keeping it inline with the actual lingo for Solana is better.

Reply:
Sure.  I am not a great namer.

Reply:
renamed in  https://github.com/LayerZero-Labs/devtools/pull/1091/commits/294318a3e59efd4374cafd69419b5ad88fd6a267